tutorials/install.md

```diff
@@ -85,6 +125,54 @@ Build and install as follows:
     make -j4
     sudo make install
 
+### Windows 10
+
+#### Prerequisites
+
+1. Install [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
+   Miniconda suffices.
+
+1. Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
+   The Community version is free for students, open-source contributors, and individuals.
+   Check "Desktop development with C++" in the "Workloads" tab, and uncheck "C++ CMake Tools". We will install `cmake` via Conda.
+
+#### Building from source
+
+Open a Visual Studio Command Prompt (search for "x64 Native Tools Command Prompt for VS 2019" in the search field near the Windows button).
+Optionally, right-click and pin to the task bar for quick access in the future.
+
+If you did not add Conda to your `PATH` environment variable during Conda installation, you may need to navigate to the location of `condabin` in order to use the `conda` command.
+To find `condabin`, search for "Anaconda Prompt" in the search field near the Windows button, open it, run `where conda`, and look for a line containing the directory `condabin`.
+
+1. Navigate to your `condabin` if necessary, and then create and activate a Conda environment:
+
+        conda create -n ign-ws
+        conda activate ign-ws
+
+   Once you have activated an environment, a prefix like `(ign-ws)` will be prepended to your prompt, and you can use the `conda` command outside of `condabin`.
+
+   You can use `conda info --envs` to see all your environments.
+
+   To remove an environment, use `conda env remove --name <env_name>`.
+
+1. Install dependencies
+
+        conda install git cmake pkg-config --channel conda-forge 
+
+1. Navigate to where you would like to build the library, and then clone the repository.
+
+        # Optionally, append `-b ign-cmake#` (replace # with a number) to check out a specific version
+        git clone https://github.com/ignitionrobotics/ign-cmake.git
+
+1. Build. Tests do not build at this time, so we turn tests off.
```

Comment:
Do we know why tests aren't building with this setup? They're building and passing on Windows CI:

https://build.osrfoundation.org/job/ignition_cmake-ci-pr_any-windows7-amd64/405/consoleFull#console-section-7

Reply:
I don't know the real reason. I followed the tip from @traversaro here https://github.com/ignitionrobotics/docs/issues/96#issuecomment-723648852
If I don't specify `-DBUILD_TESTING=OFF`, then I get errors like this from a file within the `test` directory:

```
>cmake --build .
Microsoft (R) Build Engine version 16.8.2+25e4d540b for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  nonvirtual_destructor.cc
Z:\sim\ign_ws\src\ign-cmake\test\warning\nonvirtual_destructor.cc(28,3): error C2220: the follow
ing warning is treated as an error [Z:\sim\ign_ws\build\ignition-cmake2\test\warning\WARNING_non
virtual_destructor.vcxproj]
Z:\sim\ign_ws\src\ign-cmake\test\warning\nonvirtual_destructor.cc(28,3): warning C5205: delete o
f an abstract class 'Base' that has a non-virtual destructor results in undefined behavior [Z:\s
im\ign_ws\build\ignition-cmake2\test\warning\WARNING_nonvirtual_destructor.vcxproj]
Z:\sim\ign_ws\src\ign-cmake\test\warning\nonvirtual_destructor.cc(6): message : see declaration
of 'Base' [Z:\sim\ign_ws\build\ignition-cmake2\test\warning\WARNING_nonvirtual_destructor.vcxpro
j]
Z:\sim\ign_ws\src\ign-cmake\test\warning\nonvirtual_destructor.cc(28,3): message : to simplify m
igration, consider the temporary use of /Wv:18 flag with the version of the compiler with which
you used to build without warnings [Z:\sim\ign_ws\build\ignition-cmake2\test\warning\WARNING_non
Virtual_destructor.vcxproj]
```

Reply:
Hmm, I'm not running into this error and tests are passing for me, not sure why this may be occurring.  I'll probably go ahead and add tests to this tutorial and just have a note indicating that you may need to turn off tests with that flag if you run into issues with them.

Reply:
Hmm, I wonder if we have the same CMake versions - would that make a difference?

Reply:
I moved the documentation and testing sections to the readme, it seems a little odd to me that those would reside in the tutorial for installing so I just moved the more information documentation and testing here to the readme d88bce53d207b2ba6b2fcc90bd750950771803d8